<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="3">
     <widget class="QWidget" name="widget_2" native="true">
      <widget class="QPushButton" name="HybirdBtn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>101</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Hybird</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <widget class="QPushButton" name="ChalmerBtn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Chalmer</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="ChalmerGraphic"/>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="HybirdGraphic"/>
    </item>
    <item row="3" column="0">
     <widget class="QGraphicsView" name="PivotGraphic"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QWidget" name="widget_3" native="true">
      <widget class="QPushButton" name="PivotBtn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Pivot</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QPushButton" name="closeBtn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
